### PR TITLE
Create ios-build.yaml

### PR DESCRIPTION
**Pull request by mbirghan**

### CODE REVIEW

File: .github/workflows/ios-build.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+jobs:
+  macos:
+    name: iOS build
+    runs-on: macos-latest
+
+    strategy:
+      matrix:
+        config:
+            - { name: "Qt5", pattern: qt5, folder: ios }
```
Comment (bakulf) [inline .github/workflows/ios-build.yaml:25]: Qt6 soon, right?

**Review by bakulf — APPROVED** [review]

**Comment by mbirghan** [comment]
Yes! Qt6 should be ready as soon as the static packages are uploaded to the repo. 
I uploaded some packages today but apparently there were some files missing so I will fix that next week.

**Comment by bakulf** [comment]
Can you name them qt6*?
If you need help i can tell you how I made the initial qt5 files.
A combination of `tar cfj` and `split`

**Comment by mbirghan** [comment]
I already did that. You can see it on the qt_ios repository. 
But somehow some files are missing.

**Pull request merged**